src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -160,5 +160,29 @@
             <argument type="service" id="translator" />
             <tag name="kernel.cache_warmer" />
         </service>
+
+        <service id="translation.translator" class="Symfony\Component\Translation\Translator">
+            <argument>%kernel.debug%</argument>
+            <argument type="service" id="translation.catalogue_bag.cache" />
```

Comment:
Could be nice to have an option to change the default `CatalogueBag`. For instance with openl10n/openl10n-translation I don't use the translations from the resource files but from a database. That's the point of this PR to ease the use of a different MessageCatalogue provider.

I think you can use here an alias `translation.catalogue_bag.default` (or `translation.message_catalogue_provider`), and use the Configuration to specify it. For instance:

```yaml
framework:
  translation:
    message_catalogue_provider: translation.message_catalogue_provider.cache # by default
```